docs/deployment/version_skew_policy.md

```diff
@@ -61,21 +71,35 @@ Prerequisites:
 
 - In a single-instance setup, the existing `gardener-apiserver` instance is **1.37**.
 - In a multi-instance setup, all `gardener-apiserver` instances are at **1.37** or **1.38** (this ensures maximum skew of 1 minor version between the oldest and newest `gardener-apiserver` instance).
-- The `gardener-controller-manager`, `gardener-scheduler`, `gardener-admission-controller`, and `gardenlet` instances that communicate with this `gardener-apiserver` are at version **1.37** (this ensures they are not newer than the existing API server version and are within 1 minor version of the new API server version).
+- The `gardener-controller-manager`, `gardener-scheduler`, and `gardener-admission-controller`
+- `gardenlet` instances on all seeds are at version **1.37** or **1.36** (this ensures they are not newer than the existing API server version and are within 2 minor versions of the new API server version).
```

Comment:
It seems like there is something missing in the first line, e.g. `instances that communicate with ...`. I assume that we still want to specify, which version those components should be at.

Reply:
Oops, not sure how this got lost. Thank you!